_services_projects/forest-service.md

```diff
@@ -20,50 +20,25 @@ product_clients:
 [Acquisition’s Agile Blanket Purchase Agreement](https://18f.gsa.gov/what-we-deliver/agile-bpa/) (BPA).
 
-Instead of putting the entire permit application tool into one RFP, the
-team started by asking only for an API to connect to a legacy system
-within the Forest Service. By breaking the work into pieces (called
-modules) the Forest Service was able to get functionality quickly while
-reducing risk on each module.
+Instead of putting the entire permit application tool into one RFP, we started by building an API to connect to a legacy system within the Forest Service. By breaking the work into pieces (called modules) the Forest Service was able to get functionality quickly while reducing risk on each module.
 
-Working with the Forest Service and the vendor, we built the first API
-module in four months. We provided technical and design guidance to the
-vendor, and strategies for the Forest Service to serve as empowered
-product owners.
+Working with the Forest Service and the vendor, we built the initial API module in four months. We provided technical and design guidance to the vendor as well as coaching and strategies for the Forest Service to serve as product owners.
 
-With the API module in hand and working well, we then helped the Forest
-Service draft, award, and manage two additional RFPs to build a
-public-facing application and a module to support Christmas Tree
-permits. We also integrated the application with
-[login.gov](http://login.gov) so users could securely log in to the
-system.
+With the API module in hand and working well, we then helped the Forest Service draft, award, and manage additional RFPs to build a public-facing application, a module to support Christmas tree permits, and a module to support Special Uses permits. We also integrated the application with [login.gov](http://login.gov) so users could securely log in to the system and [Pay.gov](https://pay.gov) for credit card payment processing.
 
-Breaking the system into modules and using 18F’s Agile BPA allowed the
-Forest Service to see results quickly, reduce risk on each module, and
-deliver a service that meets the needs of users. The 18F acquisition
-team used each module to help the Forest Service learn new software
-development and contract management techniques that will help them
-continue to be a better buyer of technology.
+In 2018, the system received its ATO, and we deployed the Christmas tree module into production for four pilot forests so we can learn how the product works for its users. From here, we will use learnings to inform a scaling strategy. 
+
+Breaking the system into modules and using 18F’s Agile BPA allowed the Forest Service to see results quickly, reduce risk on each module, and deliver a service that meets the needs of users. The 18F acquisition team used each module to help the Forest Service learn new software development and contract management techniques that will help them continue to be a better buyer of technology.
```

Comment:
* "From here, we will use learnings to inform a scaling strategy." I wonder if we want to highlight that we'll be providing product and strategic support for the Forest Service to implement their own scaling strategy.
* instead of "using 18F’s Agile BPA" we might want to say "modular contracts"